lib/ansible/modules/network/junos/junos_netconf.py

```diff
@@ -139,7 +139,7 @@ def map_params_to_obj(module):
     for key, value in iteritems(obj):
         # validate the param value (if validator func exists)
         validator = globals().get('validate_%s' % key)
-        if all((value, validator)):
+        if callable(validator):
```

Comment:
Well, `netconf_port` has a default, so it shouldn't be `None` anyway, which means the value guarding is unnecessary anyway (and harmful if `netconf_port == 0`)

I also freely admit that querying `globals()` to find function names is more extensible, but it is so at the cost of readability and surprise factor when trying to debug. And since the methods are manually authored instead of dynamically imported or generated, it is easy to save someone else the cognitive load of wondering what might get a validator function & from where by spelling them out rather than letting Python find them for you. But that's only my opinion and I don't think that's enough to hold this back if you don't agree.